godtools/App/Share/Views/VideoPlayerView/VideoPlayerView.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  VideoPlayerView.swift
+//  godtools
+//
+//  Created by Robert Eldredge on 11/1/21.
+//  Copyright © 2021 Cru. All rights reserved.
+//
+
+import UIKit
+import youtube_ios_player_helper
+
+class VideoPlayerView: UIViewController {
```

Comment:
Hey @reldredge71 something I'd like to see on this view is pausing and resuming the video when the viewController life cycle methods are invoked.  Just to ensure the video won't be playing if something were presented over this view or if we were to navigate to a view while this view is still in the stack.


![view-controller-life-cycle](https://user-images.githubusercontent.com/59846460/141348442-9de7f4cb-4d24-4ffe-b231-85b675afd893.png)